src/eventer/eventer_aco_opset.c

```diff
@@ -117,15 +117,15 @@ eventer_aco_init(void) {
 eventer_aco_t
 eventer_set_eventer_aco_co(eventer_t e, aco_t *co) {
   if(co == NULL) {
-    mtevAssert(e->opset == eventer_aco_fd_opset);
+    mtevAssert((uintptr_t)e->opset == (uintptr_t)eventer_aco_fd_opset);
```

Comment:
I am assuming you are casting these for a purpose.  However, this same cast does not happen on lines 72, 89, or 480, or in the assignment on 135.  If it's important here, should it not also be important there?